### PR TITLE
Update chart properties for backup/restore with Medusa

### PR DESCRIPTION
**Pull request by jsanda**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Chart versions updated (if necessary)
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 1]
```diff
@@ -126,20 +126,21 @@ Notice how in this example, the region defined in the AWS console is `us-east-1`
...
-    storage: s3
-    storage_properties:
-      region: us-east-1
-```
+cassandra:
+  datacenters:
+  - name: dc1
+    size: 3
+medusa:
+  enabled: true
```
Comment (burmanm) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:134]: Something is wrong with this indentation?

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 2]
```diff
@@ -126,20 +126,21 @@ Notice how in this example, the region defined in the AWS console is `us-east-1`
...
+   bucketName: k8ssandra-bucket-dev
+   storageSecret: medusa-bucket-key
+   storage: s3
+   storage_properties:
+     region: us-east-1```
```
Comment (burmanm) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:139]: Is the ``` as last item intended instead of after line change?

**Comment by burmanm** [comment]
Is this missing the updates to the Charts themselves?

**Comment by burmanm** [comment]
Ah, no .. there's just so many ``bucketSecret`` references in k8ssandra code. Maybe those should be updated and removed also? There's even one test that uses it..

**Comment by jsanda** [comment]
I thought I updated/remove the `bucketSecret` references in the code. I can check and open a separate PR for that.

**Comment by burmanm** [comment]
Yeah, there's one in charts/k8ssandra/values.yaml and one in tests/unit/template_medusa_config_test.go

**Review by burmanm — APPROVED** [review]

**Pull request merged**